_posts/2016-9-2-jade-release.markdown

```diff
@@ -35,7 +35,7 @@ Among numerous bug-fixes, there are quite a number of new features included:
 ![1st Jade release](http://www.ros.org/news/assets_c/2015/05/jadeturtle_LoRes-thumb-480x519-1131.jpg)
 
 ROS Indigo
-==========
+----------
```

Comment:
i don't understand why these old blogs are being updated

Reply:
Task was:
Auto-include the page title as an <h1> at the top of every page
-using the meta data from Jekyl
-check that existing pages don't already include <h1> title at the top (some do). Remove the ones that do so there isn't duplicate:
so I changed 
======= ALT-H1 to ------- means  Alt-H2
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet

Reply:
ah, my bad.